src/NexusMods.App.UI/Resources/Language.pl.resx

```diff
@@ -139,10 +139,10 @@
 		<value>Dodaj grę</value>
 	</data>
 	<data name="DownloadsView__DOWNLOADS" xml:space="preserve">
-		<value>POBRANE</value>
+		<value>POBIERANIE</value>
```

Comment:
This should probably still say `POBRANE` as `POBERIANIE` is DOWNLOAD(ING), but this string is unused currently it seems.